modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/ScriptElementAttributesFormField.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {v4 as uuidv4} from 'uuid';
+
+import AttributeFields from './AttributesFields';
+
+const emptyRow = () => ({id: uuidv4(), name: '', type: 'String', value: ''});
+
+const ScriptElementAttributesFormField = ({
+	attributes: initialAttributes,
+	portletNamespace,
+}) => {
+	const [attributesList, settAtributesList] = useState(() =>
+		initialAttributes && !!initialAttributes.length
+			? [initialAttributes.map((item) => ({...item, id: uuidv4()}))]
+			: [emptyRow()]
+	);
+
+	const handleAddClick = (index) => {
+		settAtributesList(attributesList.toSpliced(index + 1, 0, emptyRow()));
+	};
+
+	const handleRemoveClick = (index) => {
+		settAtributesList(attributesList.toSpliced(index, 1));
+	};
+
+	const updateAttributeList = (index, updatedValue) => {
+		settAtributesList((prevList) =>
+			prevList.with(index, {...prevList[index], ...updatedValue})
+		);
+	};
+
+	const filteredList = (attributes) => {
+		const attributesObject = {};
+
+		attributes.map((attribute) => {
+			let value = attribute.value;
+
+			if (attribute.type === 'Boolean') {
+				value = JSON.parse(value);
```

Comment:
The reasodn I needed to handle the value was because it was always stored like a string, no matter the type.

```
{ 
    isBoolean: "true",
    isNull: "false"
}
```

Reply:
I updated the function to send the value as a boolean. 
```typescript
onSelectionChange={(value) =>
	(index, {value: JSON.parse(value)})
}
```

Reply:
Hey @ilzamcmed sorry I forgot that the picker only returns a text value. You can switch it back to how you had it before... :)